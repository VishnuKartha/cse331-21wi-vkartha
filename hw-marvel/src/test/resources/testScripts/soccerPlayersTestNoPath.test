# Tests the expected behavior of no path between characters

# To run this test, the soccer dataset must exist at
#   hw-marvel/src/main/resources/data/soccer.tsv

LoadGraph soccer soccer.tsv

FindPath soccer Leo_Messi Cristiano_Ronaldo
FindPath soccer Cristiano_Ronaldo Leo_Messi