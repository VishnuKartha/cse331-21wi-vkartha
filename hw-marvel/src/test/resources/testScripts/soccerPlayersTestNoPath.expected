# Tests the expected behavior of no path between characters

# To run this test, the soccer dataset must exist at
# hw-marvel/src/main/resources/data/soccer.tsv

loaded graph soccer

path from Leo Messi to Cristiano Ronaldo:
no path found
path from Cristiano Ronaldo to Leo Messi:
no path found